gift-app/database/models/models.py

```diff
@@ -11,14 +12,21 @@
 class User(Base, SerializerMixin, BaseModel):
     __tablename__ = 'user'
 
-    serialize_only = ('id', 'name', 'last_name',
+    serialize_only = ('id', 'first_name', 'last_name', 'email_address',
+                      'time_created', 'is_activated', 'is_deleted',
                       'groups.id', 'groups.name',
                       'lists.id', 'lists.name',
-                      'items.id', 'items.name')
+                      'items.id', 'items.idea')
 
     id = Column(Integer, primary_key=True)
-    name = Column(String(254))
+    first_name = Column(String(254))
     last_name = Column(String(254))
+    email_address = Column(String(254))
+    hashed_password = Column(String(254))
+    time_created = Column(DateTime(timezone=True), server_default=func.now())
+    is_activated = Column(Boolean)
```

Comment:
Can you make isavtive default to False?

Reply:
I think it's fine to have it default to False, but what are we thinking "is_activated" should be used for? My thought is when someone adds emails to invite people to a group, we'll add those emails into the db with is_activated=false. Then when people sign up we'll check for whether their email already exists in the db, & if it does, change is_activated=true while adding all the other info? But when people just sign up and their email isn't in the db yet, we'll add them with is_activated=true. Does that match your idea?